runtime/src/migrations.rs

```diff
@@ -18,6 +18,81 @@
 #[allow(unused_imports)]
 use super::*;
 
+mod migrate_change_vk_format {
+
+    use super::*;
+
+    pub struct RemoveUltraplonkVks;
+
+    #[cfg(feature = "try-runtime")]
+    type Balances = sp_std::collections::btree_map::BTreeMap<crate::AccountId, crate::Balance>;
+
+    impl RemoveUltraplonkVks {
+        #[cfg(feature = "try-runtime")]
+        fn get_all_tickets_balances() -> Balances {
+            use frame_support::traits::fungible::Inspect;
+
+            pallet_ultraplonk_verifier::Tickets::<Runtime>::iter()
+                .map(|((id, _), _)| {
+                    let balance = crate::Balances::balance(&id);
+                    (id, balance)
+                })
+                .collect::<sp_std::collections::btree_map::BTreeMap<_, _>>()
+        }
+    }
+
+    impl frame_support::traits::OnRuntimeUpgrade for RemoveUltraplonkVks {
+        #[cfg(feature = "try-runtime")]
+        fn pre_upgrade() -> Result<sp_std::vec::Vec<u8>, sp_runtime::TryRuntimeError> {
+            use codec::Encode;
+
+            frame_support::ensure!(
+                crate::System::last_runtime_upgrade_spec_version() < 10_000,
+                "must upgrade linearly"
+            );
+
+            Ok(Self::get_all_tickets_balances().encode())
+        }
+
+        /// Migrates validators and nominators to bags list for the staking pallet.
+        fn on_runtime_upgrade() -> Weight {
+            use frame_support::traits::UncheckedOnRuntimeUpgrade;
+            if crate::System::last_runtime_upgrade_spec_version() >= 10_000 {
```

Comment:
In order to speed things up I would integrate this migration in the runtime v0.10.0, so please update this number to 9000 instead

Reply:
That's ok, it check if the version is >= than 10_000 in order TO SKIP IT (to avoid issue if we forget to remove it)